content/en/platform/corda/5.0/deploying-operating/deployment/deploying/_index.md

```diff
@@ -468,14 +468,14 @@ config:
     createdSecretPath: "<path-to-corda-created-secrets>"
 ```
 
-* `<vault-URL>` is the full URL including port at which the vault instance is reachable, not including any path.
-* `<vault-token>` must allow sufficient permissions to read from vault at the Corda configured paths and write to the `<path-to-corda-created-secrets>`, where Corda writes secrets it creates.
+* `<vault-URL>` is the full URL including port at which the Vault instance is reachable, not including any path.
+* `<vault-token>` must allow sufficient permissions to read from Vault at the Corda configured paths and write to the `<path-to-corda-created-secrets>`, where Corda writes secrets it creates.
 
-The passwords for the {{< tooltip >}}RBAC{{< /tooltip >}} and CRYPTO schemas and VNODES database must be available in vault before Corda is deployed. These must be available in the Vault `dbsecrets` path, under the keys `rbac`, `crypto`, and `vnodes` respectively.
+The passwords for the `RBAC` and `CRYPTO` schemas and `VNODES` database must be available in Vault before Corda is deployed. These must be available in the Vault `dbsecrets` path, under the keys `rbac`, `crypto`, and `vnodes` respectively.
```

Comment:
I know it's not something you've changed but I suspect that `dbsecrets` here is whatever you've provided for `createdSecretPath`.

@dickon - can you confirm?